### DISCUSSION TITLE
Is a Wort zum Sonntag a thing?

### DISCUSSION THREAD

**Discussion by riesentoaster** [riesentoaster/geiman01#7] [Thingification]
title

**Comment by floomm** [comment]
Everything is a thing. That's why it's called everything.

**Reply by riesentoaster** [reply]
Yes, `everything` is a thing indeed. But there most certainly are nothings. Like `nothing`. `nothing` is a nothing. And then, there are maybethings. Where we don't know yet.

Also: What about a Wort zum Sonntag?

**Reply by floomm** [reply]
Is nothing really nothing?

**Reply by riesentoaster** [reply]
So you'd argue it should be a maybething?

**Reply by Geissemanu** [reply]
Nothing is a thing but there are things that are nothings as not everything is a thing... 
It's not that hard

**Reply by riesentoaster** [reply]
Please don't talk in absolutes when we are not yet sure about this. This stifles the discourse and diametrically opposes our discussion culture.

**Reply by Geissemanu** [reply]
only a sith talks in absolutes...
